dev/com.ibm.ws.logging.osgi/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -14,76 +14,73 @@
+If you specify a value of true, the ISO-8601 format is used in the messages.log file, the trace.log file, and the FFDC logs. The format is yyyy-MM-dd'T'HH:mm:ss.SSSZ. \
+If you specify a value of false, the date and time are formatted according to the default locale set in the system. If the default locale is not found, the format is dd/MMM/yyyy HH:mm:ss:SSS z.
 
 filter.sensitive=Suppress sensitive trace
 filter.sensitive.desc=The server trace can expose sensitive data when tracing untyped data, such as bytes received over a network connection. \
-If true, prevent potentially sensitive information from being exposed in log and trace files. The default value is false.
+If true, prevent potentially sensitive information from being exposed in log and trace files. The default value is false. Avoid trouble: Use the com.ibm.ws.logging.filter.sensitive property, except in cases where you are trying to change the configuration dynamically after startup.
 
 hide.messages=Messages to be hidden
-hide.messages.desc=The list of messages, separated by a comma, that are configured to be hidden from the console.log and message.log files. If the messages are configured to be hidden, then they are redirected to the trace.log file.
+hide.messages.desc=The list of messages, separated by a comma, that are configured to be hidden from the console.log and message.log files. If the messages are configured to be hidden, then they are redirected to the trace.log file. Avoid trouble: Use the com.ibm.ws.logging.hide.messages property, except in cases where you are trying to change the configuration dynamically after startup.
 
 message.source=Message source
-message.source.desc=The list of sources to route to messages.log
+message.source.desc=The list of comma-separated sources that route to the messages.log file. This property only applies when messageFormat=json. Valid values are message, trace, accessLog, ffdc, and audit. By default, messageSource is set to the environment variable WLP_LOGGING_MESSAGE_SOURCE (if set), or message. Note: To use the audit source, enable the Liberty audit feature. Avoid trouble: Use the WLP_LOGGING_MESSAGE_SOURCE environment variable or com.ibm.ws.logging.message.source property, except in cases where you are trying to change the configuration dynamically after startup.
 
 message.format=Message format
-message.format.desc=The format to use for messages.log
+message.format.desc=The required format for the messages.log file. Valid values are basic or json format. By default, messageFormat is set to the environment variable WLP_LOGGING_MESSAGE_FORMAT (if set) or basic. Avoid trouble: Use the WLP_LOGGING_MESSAGE_FORMAT environment variable or com.ibm.ws.logging.message.format property, except in cases where you are trying to change the configuration dynamically after startup.
```

Comment:
Change 
"Valid values are basic or json format. By default, messageFormat is set to the environment variable WLP_LOGGING_MESSAGE_FORMAT (if set) or basic. Avoid trouble: Use the WLP_LOGGING_MESSAGE_FORMAT environment variable or com.ibm.ws.logging.message.format property, except in cases where you are trying to change the configuration dynamically after startup."
to
"Valid values are basic or JSON format. By default, messageFormat is set to the environment variable WLP_LOGGING_MESSAGE_FORMAT (if set), or basic. Avoid trouble: Use the WLP_LOGGING_MESSAGE_FORMAT environment variable or the com.ibm.ws.logging.message.format property, except in cases where you are trying to change the configuration dynamically after startup."